src/Slider.jsx

```diff
@@ -70,7 +70,7 @@ export default React.createClass({
       algorithm: linear,
       className: '',
       disabled: false,
-      handle: 'div',
+      handle: 'button',
```

Comment:
since the default "type" of `<button>` is "submit", we need to ensure this uses an explicit type of "button".